rotkehlchen/db/accounting_rules.py

```diff
@@ -7,11 +8,22 @@
 from rotkehlchen.db.constants import NO_ACCOUNTING_COUNTERPARTY
 from rotkehlchen.db.filtering import AccountingRulesFilterQuery
 from rotkehlchen.errors.misc import InputError
+from rotkehlchen.logging import RotkehlchenLogsAdapter
 
 if TYPE_CHECKING:
     from rotkehlchen.db.dbhandler import DBHandler
 
 
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class RuleInformation(NamedTuple):
+    identifier: tuple[int, HistoryEventType, HistoryEventSubType, str]
+    rule: BaseEventSettings
+    overwrites: dict[str, Any]
```

Comment:
the overwrites term is confusing as I wrote in the function below. Please rename.